src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/IVSDidOpenTextDocumentEndpoint.cs

```diff
@@ -1,14 +1,12 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT license. See License.txt in the project root for license information.
 
-using System;
 using Microsoft.CommonLanguageServerProtocol.Framework;
 using Microsoft.VisualStudio.LanguageServer.Protocol;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts;
 
 [LanguageServerEndpoint(Methods.TextDocumentDidOpenName)]
-internal interface IVSDidOpenTextDocumentEndpoint : IRazorNotificationHandler<DidOpenTextDocumentParams>,
-    ITextDocumentIdentifierHandler<DidOpenTextDocumentParams, Uri>
```

Comment:
`textDocument/didOpen` requests were creating a logged warning because there was an attempt to look up a DocumentContext that could not be found. In this case the document was known about but didn't have a version because it hadn't been opened yet (which AFAIK is the correct behavior. Therefore I decided that the best thing to do is not to try to look up this `DocumentContext` in the first place since it's not needed. That can be accomplished by removing `ITextDocumentIdentifierHandler`.